docs/topics/link-extractors.rst

```diff
@@ -10,13 +10,21 @@ The ``__init__`` method of
 :class:`~scrapy.linkextractors.lxmlhtml.LxmlLinkExtractor` takes settings that
 determine which links may be extracted. :class:`LxmlLinkExtractor.extract_links
 <scrapy.linkextractors.lxmlhtml.LxmlLinkExtractor.extract_links>` returns a
-list of matching :class:`scrapy.link.Link` objects from a
+list of matching :class:`~scrapy.link.Link` objects from a
 :class:`~scrapy.http.Response` object.
 
 Link extractors are used in :class:`~scrapy.spiders.CrawlSpider` spiders
 through a set of :class:`~scrapy.spiders.Rule` objects. You can also use link
 extractors in regular spiders.
 
+In order to use link extractors in regular spiders, you can instantiate ``scrapy.linkextractors.LinkExtractor`` into
+a class variable in your spider. This variable can be accessed through the ``parse`` method
+and used to extract links using a format similar to the one below::
+
+    def parse(self, response):
+        for link in self.link_extractor.extract_links(response):
+            yield Request(link.url, callback=self.parse)
+
```

Comment:
:lipstick: What do you think about moving the last sentence above into your new paragraph, and
phrasing it as an example (since this is just _one_ possible approach):

```suggestion
through a set of :class:`~scrapy.spiders.Rule` objects.

You can also use link extractors in regular spiders. For example, you can instantiate
:class:`LinkExtractor <scrapy.linkextractors.lxmlhtml.LxmlLinkExtractor>` into a class
variable in your spider, and use it from your spider callbacks::

    def parse(self, response):
        for link in self.link_extractor.extract_links(response):
            yield Request(link.url, callback=self.parse)

```

Reply:
Looks good. I just applied the suggestion. Since there are other ways to use them, I could try looking them up (I don't know them at the moment) and also add them.

Reply:
I think a single example is enough. I meant that users, for example, do not have to keep the link extractor object in the class, and they can use them outside callbacks. You cover the most, which I think is best when providing an example while keeping things short.